orthos2/api/commands/delete.py

```diff
@@ -90,6 +92,12 @@ def get(self, request, *args, **kwargs):
 
             return redirect(reverse('api:remotepower_delete'))
 
+        elif item == Delete.REMOTEPOWERDEVICE:
+            if sub_arguments:
+                return ErrorMessage("Invalid number of arguments for 'remotepowerdevice'!").as_json
```

Comment:
Better try to avoid extra chars. Especially single quote and Exclamation Mark are evil.
Imagine you want to parse this at some point of time.
why not use: [remotepowerdevice] (or similar)?